test.js

```diff
@@ -0,0 +1,76 @@
+const windows = require('./')('1.8')
```

Comment:
I see there is a test.js now...

would just need to convert it to mocha then

Reply:
made tests with jest because mocha didn't work for me, hope thats fine.

Reply:
You forgot to commit

Reply:
no, i did not.
now that the pr is closed i guess that commits to my forked repo don't show up anymore, but they can be seen here:
https://github.com/KadaverBrutalo10/prismarine-windows